policies/gitlab/member.rego

```diff
@@ -34,11 +34,11 @@ two_factor_authentication_is_disabled_for_a_collaborator {
 #   threat: 
 #     - "Collaborators without two-factor authentication are prime targets for phising and social engineering attacks, as compromise only requires acquiring the collaborator's password."
 #     - "This is doubly important for external collaborators, as these are identities that aren't likely managed by you or your organization and may be easier to compromise."
-default two_factor_authentication_is_disabled_for_an_external_collaborator = false
+default two_factor_authentication_is_disabled_for_an_external_collaborator = true
 
-two_factor_authentication_is_disabled_for_an_external_collaborator {
-	input.external == true
-	input.two_factor_enabled == false
+two_factor_authentication_is_disabled_for_an_external_collaborator = false {
+	not input.external 
```

Comment:
need to revert this line since this policy should be tested for external collaborators